split-webapp/split/src/components/TransactionList.js

```diff
@@ -13,14 +13,34 @@ import "../App.css";
 function ListItemLink(props) {
   return <ListItem button component="a" {...props} />;
 }
- 
-export default function TransactionList() {
 
-const transaction = {
+class TransactionList extends React.Component {
```

Comment:
Glad to see that we're using React Components now - is this applicable to all future components?

Reply:
Yeah we will be doing this from now one!